rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/EvaluatorUtil.java

```diff
@@ -248,15 +248,22 @@ public static CompletableFuture<Evaluator> makeFutureEvaluator(ExecutorService e
                     eval.addRascalSearchPath(URIUtil.correctLocation("lib", "rascal-core", ""));
                 }
                 eval.addRascalSearchPath(URIUtil.correctLocation("lib", "rascal-lsp", ""));
-
+                
                 if (pcfg != null) {
                     for (IValue src : pcfg.getSrcs()) {
                         eval.addRascalSearchPath((ISourceLocation) src);
                     }
                 }
 
-                eval.doImport(services, imports);
-
+                for (String i : imports) {
```

Comment:
The merge went wrong here, this was the old code, that got removed in this commit: https://github.com/usethesource/rascal-language-servers/commit/dae2ab7eb871c71fe4dda0c668e17250b413f406